flang/include/flang/Runtime/command.h

```diff
@@ -47,6 +47,12 @@ std::int32_t RTNAME(GetEnvVariable)(const Descriptor &name,
     bool trim_name = true, const Descriptor *errmsg = nullptr,
     const char *sourceFile = nullptr, int line = 0);
 }
+
+// Try to get the name of current user
+// Returns a STATUS as described in the standard.
+std::int32_t RTNAME(GetLog)(
+    const Descriptor *argument = nullptr, const Descriptor *errmsg = nullptr);
+
```

Comment:
I believe all the runtime entry point of command.h are here to cover standard compliant intrinsic (and that calls to them may be generated by the compiler). 

I am not very opposed to defining this entry point, but do you see a use case outside of the non standard `getlog`?

Reply:
Sorry I couldn't understand.. Are you suggesting having it as a lowing intrinsics? If I don't define it here, then under `flang/runtime/command.cpp`, it won't know what `RTNAME(GetLog)` is, because it is not defined.

Reply:
It has been moved to extension.cpp where the implementation is inline. I'm not sure if we need the right trim there.